app/Models/Department.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Department extends Model
+{
+    use HasFactory;
+
+    public function users()
+    {
+        return $this->hasMany(User::class);
```

Comment:
Make a habit of define the foriegn key as second param.  The way you become clear about how relationship mapping is executed behind the scene.

So the better way would be 
        `return $this->hasMany(User::class, "user_id", "id");`